pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -412,6 +412,58 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: PersistentVolumeControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				// PersistentVolumeBinder.volumeController.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("persistentvolumes"),
+				},
+				// PersistentVolumeBinder.syncVolume()
+				{
+					Verbs:     sets.NewString("get", "update", "create", "delete"),
```

Comment:
I arranged them this way just to avoid duplication.

Both the binder and recycler require Get/Update.  Recycler requires Create/Delete.  They are all in one here, since it's the same account.